<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetup</class>
 <widget class="QDialog" name="DialogSetup">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_udp">
     <property name="title">
      <string>UDP</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_udpAddress"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_udpPort">
        <property name="text">
         <string>Port</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_udpPort</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_udpAddress">
        <property name="text">
         <string>Host address</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_udpAddress</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_udp">
        <property name="text">
         <string>UDP Listener</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_udpPort"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_Station">
     <property name="title">
      <string>Station</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_qthLocator">
        <property name="text">
         <string>QTH Locator</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_qthLocator</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_qthLocator"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_Tracking">
     <property name="title">
      <string>Tracking path</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_AirScout">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_AirScout"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_AirScout">
        <property name="text">
         <string>AirScout</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_AirScout</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_WSJTX">
        <property name="text">
         <string>WSJT-X</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_WSJTX</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_WSJTX"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_WSJTX">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_qthLocator</tabstop>
  <tabstop>checkBox_udp</tabstop>
  <tabstop>lineEdit_udpAddress</tabstop>
  <tabstop>lineEdit_udpPort</tabstop>
  <tabstop>lineEdit_WSJTX</tabstop>
  <tabstop>pushButton_WSJTX</tabstop>
  <tabstop>lineEdit_AirScout</tabstop>
  <tabstop>pushButton_AirScout</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
